frontend/package.json

```diff
@@ -4,15 +4,17 @@
   "private": true,
   "scripts": {
     "dev": "vite",
-    "build": "vite build",
+    "build": "yarn configure && vite build",
```

Comment:
The reason why it is added in the `build` process is so that the app fails to build and deploy when an error was thrown by the config generation script.

I tested the approach of having it only on `postinstall` and that is not stopping the build & deployment since they are 2 different processes (the `install/postinstall` & `build`). Therefore this is necessary.

Reply:
We could also modify the CI to make it fail in case the postinstall failed. Or modify the vite build. But I think this is going too far now. So let's leave it.